src/dev/app.md

```diff
@@ -29,15 +31,63 @@ docker pull cozy/cozy-app-dev
 
 (We update this image on a regular basis with the latest version of the server and our library. Don’t forget to update the image by running `docker pull cozy/cozy-app-dev` from time to time).
 
+## Create your application
 
-## Create your first application
+You can boostrap your application from scratch if you want, but we recommand to use our new community tool [`create-cozy-app`](https://github.com/CPatchane/create-cozy-app) to bootstrap very easily a Cozy application for you.
```

Comment:
"community tool", such a political term that have nothing to do in a getting started documentation, imho.